src/jit-analyze/jit-analyze.cs

```diff
@@ -432,54 +427,44 @@ public static int Summarize(IEnumerable<FileDelta> fileDeltaList, Config config,
 
-            if (methodRegressionCount > 0)
-            {
-                Console.WriteLine("\nTop method regressions by size (bytes):");
+            var sortedMethodImprovementsByPercentage = methodDeltaList
+                                            .Where(x => x.deltaBytes < 0)
+                                            .OrderBy(d => (double)d.deltaBytes / d.baseBytes).ToList();
+            var sortedMethodRegressionsByPercentage = methodDeltaList
+                                            .Where(x => x.deltaBytes > 0)
+                                            .OrderByDescending(d => (double)d.deltaBytes / d.baseBytes).ToList();
 
-                foreach (var method in sortedMethodRegressions.GetRange(0, Math.Min(methodRegressionCount, requestedCount)))
+            void DisplayMethodMetric(string headerText, int metric, dynamic list)
+            {
+                if (metric > 0)
                 {
-                    Console.Write("    {2,8} : {0} - {1}", method.path, method.name, method.deltaBytes);
-
-                    if (method.baseCount == method.diffCount)
+                    Console.WriteLine(headerText);
+                    foreach (var method in list.GetRange(0, Math.Min(metric, requestedCount)))
                     {
-                        if (method.baseCount > 1)
+                        Console.Write("    {2,8} ({3,6:P} of base) : {0} - {1}", method.path, method.name, method.deltaBytes,
+                            (double)method.deltaBytes / method.baseBytes);
+
+                        if (method.baseCount == method.diffCount)
                         {
-                            Console.Write(" ({0} methods)", method.baseCount);
+                            if (method.baseCount > 1)
+                            {
+                                Console.Write(" ({0} methods)", method.baseCount);
+                            }
                         }
-                    }
-                    else
-                    {
-                        Console.Write(" ({0}/{1} methods)", method.baseCount, method.diffCount);
-                    }
-
-                    Console.WriteLine();
-                }
-            }
-
-            if (methodImprovementCount > 0)
-            {
-                Console.WriteLine("\nTop method improvements by size (bytes):");
-
-                foreach (var method in sortedMethodImprovements.GetRange(0, Math.Min(methodImprovementCount, requestedCount)))
-                {
-                    Console.Write("    {2,8} : {0} - {1}", method.path, method.name, method.deltaBytes);
-
-                    if (method.baseCount == method.diffCount)
-                    {
-                        if (method.baseCount > 1)
+                        else
                         {
-                            Console.Write(" ({0} methods)", method.baseCount);
+                            Console.Write(" ({0}/{1} methods)", method.baseCount, method.diffCount);
```

Comment:
This almost looks like a ratio. I think we can afford to be a bit more verbose here and have `Console.Write(" ({0} base, {1} diff methods)"` (or something similar)